src/engine/renderer/tr_local.h

```diff
@@ -3150,6 +3150,8 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 	void    R_InitImages();
 	void    R_ShutdownImages();
 
+	int R_FindImageLoader( char *filename );
```

Comment:
These functions are only used in the same file, so you can remove them from the header and declare them `static`.

Reply:
The declaration without `prefix` will be used from somewhere else by code in #159 
I may remove the other one with the `prefix`.

Reply:
I'll made it exportable later, when needed.